app/services/content_item_publisher.rb

```diff
@@ -1,10 +1,48 @@
 class ContentItemPublisher
   def publish(flow_presenters)
+    find_smart_answers_without_start_pages(flow_presenters)
     flow_presenters.each do |smart_answer|
-      content_item = FlowContentItem.new(smart_answer)
-      Services.publishing_api.put_content(content_item.content_id, content_item.payload)
-      Services.publishing_api.publish(content_item.content_id, 'minor')
+      content_item = assign_content_item(smart_answer)
+      if content_item
+        Services.publishing_api.put_content(content_item.content_id, content_item.payload)
+        Services.publishing_api.publish(content_item.content_id, 'minor')
+      end
     end
+    republish_start_page
+  end
+
+  def find_smart_answers_without_start_pages(flow_presenters)
+    @smart_answer_without_start_page = flow_presenters.select { |smart_answer| smart_answer.slug == "part-year-profit-tax-credits/y" }.first
+    @start_page_smart_answer = flow_presenters.select { |smart_answer| smart_answer.slug == "part-year-profit-tax-credits" }.first
+  end
+
+  def assign_content_item(smart_answer)
+    case smart_answer
```

Comment:
I am not sure why this exists but from the other example that I have given, this can be made redundant. 

```ruby

if smart_answer.start_page? 
...
else
...
end
```